client/showid.lua

```diff
@@ -0,0 +1,55 @@
+function DrawPlayerName(x, y, z, name)
+    local onScreen, _x, _y = GetScreenCoordFromWorldCoord(x, y, z)
+    local px, py, pz = table.unpack(GetFinalRenderedCamCoord())
+    local dist = #(vector3(px, py, pz) - vector3(x, y, z))
+
+    local scale = ((1/dist)*2)*((1/GetGameplayCamFov())*100)
+
+    if onScreen then
+        SetTextScale(0.0*scale, 0.5*scale)
+        SetTextFont(0)
+        SetTextProportional(1)
```

Comment:
This takes a boolean, not a number, in this case it needs to be true

Reply:
Removed that file